### PR TITLE
ci: workaround MSVC+release+x86 build problems

### PR DESCRIPTION
**Pull request by coryan**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/5743)

### CODE REVIEW

**Comment by coryan** [comment]
This built cleanly in my manual test.

File: ci/kokoro/windows/build-cmake.ps1 [hunk 1]
```diff
@@ -56,7 +56,11 @@ $workaround_targets=(
     # Failed around 2020-07-29
     "storage_internal_tuple_filter_test",
     # Failed around 2020-08-10
-    "storage_well_known_parameters_test"
+    "storage_well_known_parameters_test",
+    # Failed around 20201-01-25
```
Comment (devbww) [inline ci/kokoro/windows/build-cmake.ps1:60]: s/20201/2021/

File: google/cloud/bigtable/internal/common_client_test.cc [hunk 2]
```diff
@@ -37,7 +37,9 @@ class OutstandingTimersTest : public ::testing::Test {
 };
 
 TEST_F(OutstandingTimersTest, Trivial) {
-  std::make_shared<OutstandingTimers>(std::make_shared<CompletionQueue>(cq_));
+  auto unused = std::make_shared<OutstandingTimers>(
+      std::make_shared<CompletionQueue>(cq_));
+  unused.reset();
```
Comment (devbww) [inline google/cloud/bigtable/internal/common_client_test.cc:42]: Was giving the value a name enough, or did you find you needed the explicit `reset()`?

**Review by devbww — APPROVED** [review]

File: google/cloud/bigtable/internal/common_client_test.cc (hunk 2 above)
Comment (devbww) [reply google/cloud/bigtable/internal/common_client_test.cc:42]: Perhaps a comment either way would leave a breadcrumb.

File: ci/kokoro/windows/build-cmake.ps1 (hunk 1 above)
Comment (coryan) [reply ci/kokoro/windows/build-cmake.ps1:60]: Fixed.

File: google/cloud/bigtable/internal/common_client_test.cc (hunk 2 above)
Comment (coryan) [reply google/cloud/bigtable/internal/common_client_test.cc]: The reset was not needed for MSVC, let's see what the other compilers say.  I added a comment.

File: ci/kokoro/windows/build-cmake.ps1 (hunk 1 above)
Comment (devbww) [reply ci/kokoro/windows/build-cmake.ps1:60]: Don't see it.

Comment (coryan) [reply ci/kokoro/windows/build-cmake.ps1]: Argh, missed that file.

**Review by devbww — APPROVED** [review]

**Pull request merged**